sos/report/plugins/rhc.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2023 Red Hat, Inc., Jose Castillo <jcastillo@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class Rhc(Plugin, RedHatPlugin):
+
+    """
+    This plugin collects configuration of the RHC package.
+    """
```

Comment:
Since this is what gets printed when using `sos help report.plugins.rhc`, let's flesh this out a bit further - what you have in the commit message is actually great for using in the docstring.